lib/mindwendel_web/templates/admin/brainstorming/_form.html.leex

```diff
@@ -2,6 +2,13 @@
   <div class="form-group mb-2 mt-2">
     <%= label f, :name %>
     <%= text_input f, :name, class: "form-control" %>
+    <br />
+    <%= checkbox f, :option_show_link_to_settings %>
```

Comment:
Please integrating bootstrap correctly, see https://getbootstrap.com/docs/5.0/forms/checks-radios/#checks

Consider adding the corresponding css classes.

The checkbox and text are inconveniently positioned on mobile.

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/592424/111077383-de358900-84f0-11eb-8147-741c428fd058.png">

Reply:
good point, thanks